browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -240,15 +269,21 @@ Profile* SpeedreaderTabHelper::GetProfile() const {
 }
 
 void SpeedreaderTabHelper::ShowBubble(bool is_bubble_speedreader) {
-  auto* contents = web_contents();
-  Browser* browser = chrome::FindBrowserWithWebContents(contents);
-  DCHECK(browser);
-  speedreader_bubble_ =
-      static_cast<BraveBrowserWindow*>(browser->window())
-          ->ShowSpeedreaderBubble(this, is_bubble_speedreader);
+  // auto* contents = web_contents();
```

Comment:
remove comments

Reply:
I've added this logic back in and the new web ui bubble should show if feature flag is enabled